ws-nextjs-app/cypress/support/helpers/runTestsForPage.ts

```diff
@@ -4,10 +4,22 @@ export default ({
   beforeAll = [],
   testIsolation = false,
 }) => {
-  testSuites.forEach(testData => {
+  const serviceToRun = Cypress.env('ONLY_SERVICE');
+
+  let testSuitesToRun = testSuites;
+  if (serviceToRun) {
+    testSuitesToRun = testSuites.filter(
+      ({ service }) => service === serviceToRun,
+    );
+  } else {
+    testSuitesToRun = testSuites;
+  }
```

Comment:
You don't need this, since you've already assigned it on line 9.
```suggestion
  }
```